src/Components/test/testassets/BasicTestApp/wwwroot/index.html

```diff
@@ -25,22 +25,8 @@
         function navigationManagerNavigate() {
             Blazor.navigateTo('/subdir/some-path');
         }
-
-        (function () {
-            // Load either blazor.webassembly.js or blazor.server.js depending
-            // on the hash part of the URL. This is just to give a way for the
-            // test runner to make the selection.
-            if (location.hash === '#server') {
```

Comment:
What's the updated way to launch from VS with server-side execution?

Reply:
You just F5 and you will be presented with an html page with links to each scenario that has a custom startup

Reply:
OK, that sounds great. Thanks!